test/FunctionalTests/Tests/DataOData/Tests/OData.Reader.Tests/MessageToObjectModelReader.cs

```diff
@@ -615,7 +627,7 @@ private ODataBatchOperation GetRequestOperation(ODataBatchReaderTestWrapper batc
                 {
                     expectedPartElement = odataRequest.Body == null ? null : odataRequest.Body.RootElement;
                 }
-                
+
                 ODataBatchOperationRequestMessage requestMessage = batchReader.CreateOperationRequestMessage();

```

Comment:
# of blank line is still 1. This is just to remove the tailing whitespaces of the blank line.